src/Toolkit/F.php

```diff
@@ -573,6 +573,31 @@ public static function realpath(string $file, string $in = null)
         return $realpath;
     }
 
+    /**
+     * Returns the relative path of the file
+     * starting after $in
+     *
+     * @param string $file
+     * @param string $in
+     * @return string
+     */
+    public static function relativepath(string $file, string $in = null): string
+    {
+        if (empty($in) === true) {
+            return $file;
+        }
+
+        // windows
+        $file = str_replace('\\', '/', $file);
+        $in   = str_replace('\\', '/', $in);
+
+        if (Str::contains($file, $in) === false) {
+            return $file;
```

Comment:
I'm not sure about this fallback. Maybe the method should instead return `null` in this case. Otherwise there's the issue of leaking the full file path where a relative path was expected. That could be a security issue depending on how the method is used.

Reply:
In our case, we really need some valuable alternative. Maybe we could return just the filename in this case? I'm not sure. Both solutions seem to be based on assumptions.

Reply:
I think just the filename would be a good alternative. Better than nothing. :)